.gitlab-ci.yml

```diff
@@ -150,6 +148,22 @@ test:tools:
     - rm -rf ~/.gradle/daemon/
     - GRADLE_OPTS="-Xmx2560m" ./gradlew :unitTestTools --stacktrace --no-daemon
 
+test:kover:
+  tags: [ "runner:main" ]
+  image: $CI_IMAGE_DOCKER
+  stage: test
+  timeout: 1h
+  script:
+    - pip3 install datadog
+    - git fetch --depth=1 origin master
+    - rm -rf ~/.gradle/daemon/
+    - export DD_API_KEY=$(aws ssm get-parameter --region us-east-1 --name ci.dd-sdk-android.api_key --with-decryption --query "Parameter.Value" --out text)
+    - export DD_APP_KEY=$(aws ssm get-parameter --region us-east-1 --name ci.dd-sdk-android.app_key --with-decryption --query "Parameter.Value" --out text)
+    - CODECOV_TOKEN=$(aws ssm get-parameter --region us-east-1 --name ci.dd-sdk-android.codecov-token  --with-decryption --query "Parameter.Value" --out text)
+    - GRADLE_OPTS="-Xmx2560m" ./gradlew :koverReportAll --stacktrace --no-daemon
```

Comment:
Does `kover` task trigger a compilation?

Maybe to speed-up overall build pipeline we can use Gitlab build cache https://docs.gitlab.com/ee/ci/caching/ (we can use it between `analysis` and `test` groups)? wdyt?

Reply:
We could use the build cache, indeed I'll create a task to set this up

Reply:
Yes Kover triggers a full Debug+Release compilation